releasenotes-installscript/notes/apt-keys-improvement-87662a49cc6b226e.yaml

```diff
@@ -0,0 +1,17 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG-INSTALLSCRIPT.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
```

Comment:
I don't think we are following this in general but the syntax for this should be reStructuredText and not Markdown (e.g. code is escaped with double backticks instead of single ones IIRC)

Reply:
Good point, fixed.